docs/04_cv32a65x_design/source/mmu.rst

```diff
@@ -0,0 +1,1580 @@
+.. _CVA6_MMU:
+
+
+----------------------
+Memory Management Unit
```

Comment:
MMU in the embedded CV32A65X?
FYI, PlanV is upgrading the MMU documentation to expand with Sv39 and and Sv39x4, as part of the MMU unification.

Reply:
MMU removed from 65X DD